sdk/docs/iot/README.md

```diff
@@ -174,6 +174,126 @@ void my_telemetry_func()
 }
 ```
 
+### IoT Hub Pseudocode Implementation 

```

Comment:
What makes this Pseudocode? Snippet?

Reply:
u call it snippet int the text

Reply:
Yea that's fine wasn't exactly sure what to call it